src/components/Tags/Tags.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) DD360 and its affiliates.
+ */
+
+import { composeClasses } from 'lib/classes'
+import { fontSize } from 'lib/font'
+import DynamicHeroIcon, { IconName } from 'common/DynamicHeroIcon'
```

Comment:
I think we need to change the `DynamicHeroIcon` for a prop called `icon` like `Callout` component.

Because the `DynamicHeroIcon` is too big to include in a component